### PR TITLE
feat/ home page design

### PR DESCRIPTION
**Pull request by p-anja**

### CODE REVIEW

File: moja-kuhinjica-frontend/src/components/header/Header.tsx [hunk 1]
```diff
@@ -13,11 +16,11 @@ const Header = () => {
                     style={{ height: '100%', width: '100%' }}
```
Comment (koskeee) [inline moja-kuhinjica-frontend/src/components/header/Header.tsx:16]: Please don't use inline CSS

File: moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export const MenuItem = () => {
+    const [openDescription, setOpenDescription] = useState(false)
+
+    return (
+        <ColDiv width="420px" height="700px" alignItems="center">
+            <Image src={pic} alt="" />
+            <TitleWrapper height="20%" rotation={openDescription ? -90 : 90}>
+                <TitleLabel>Piletina u sosu od šampinjona</TitleLabel>
+                <svg
+                    onClick={() => setOpenDescription(!openDescription)}
```
Comment (koskeee) [inline moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx:23]: You should import this svg from svg file from assets. Not to inject it directly in code

File: moja-kuhinjica-frontend/src/pages/_document.tsx [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
+            }
+        } finally {
+            sheet.seal()
+        }
+    }
+    render() {
+        return (
+            <Html>
+                <Head>
+                    <link
```
Comment (koskeee) [inline moja-kuhinjica-frontend/src/pages/_document.tsx:37]: Please add title to your app, it  will look better in browser with title

File: moja-kuhinjica-frontend/src/pages/index.tsx [hunk 4]
```diff
@@ -59,8 +94,6 @@ const Home = () => {
                         <svg
```
Comment (koskeee) [inline moja-kuhinjica-frontend/src/pages/index.tsx:94]: Also please import this svg from svg file

File: moja-kuhinjica-frontend/src/components/header/Header.tsx (hunk 1 above)
Comment (p-anja) [reply moja-kuhinjica-frontend/src/components/header/Header.tsx:16]: Done.

File: moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx (hunk 2 above)
Comment (p-anja) [reply moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx:23]: But this SVG is not static. In css I change fill color, do rotation too. Would I still be able to do it if I import svg from assets?

File: moja-kuhinjica-frontend/src/pages/_document.tsx (hunk 3 above)
Comment (p-anja) [reply moja-kuhinjica-frontend/src/pages/_document.tsx:37]: Done.

**Review by koskeee — APPROVED** [review]

File: moja-kuhinjica-frontend/src/components/header/Header.tsx [hunk 5]
```diff
@@ -1,23 +1,27 @@
...
-                />
+                <LogoImage src={logo} alt="" />
             </LogoWrapper>
-            <TitleLabel>
-                Saznaj gde je najbliža
-                <TitleSpan> tvoja klopica</TitleSpan>
-            </TitleLabel>
-            <Button>Ulogujte se</Button>
+            <RowDiv alignItems="center" justifyContent="center" width="50%">
+                <Button selected={active == 1} onClick={() => setActive(1)}>
```
Comment (koskeee) [inline moja-kuhinjica-frontend/src/components/header/Header.tsx:15]: Please use === operator instead of ==

**Review by koskeee — COMMENTED** [review]
Please checkup comment about compare operator

File: moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx (hunk 2 above)
Comment (koskeee) [reply moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx:23]: Please then put this svg with rotation and fill in separate  component

File: moja-kuhinjica-frontend/src/pages/_document.tsx (hunk 3 above)
Comment (koskeee) [reply moja-kuhinjica-frontend/src/pages/_document.tsx:37]: Don't know what `Dunda` should mean

File: moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx (hunk 2 above)
Comment (p-anja) [reply moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx:23]: Done.

**Pull request merged**